### ISSUE TITLE
Regression between 1.1.25 and 1.1.26

### ISSUE THREAD

**Issue by Stexxen** [springdoc/springdoc-openapi#70]
I think there is a regression for @RequestBody between 1.1.25 and 1.1.26

For the Following
```
  @Operation(summary="test Request")
  @RequestBody(
          description = "test value", required = true,
          content = @Content(schema = @Schema(implementation = String.class)))

```

1.1.25 generates
```
      summary: test Request
      operationId: testRequest
      requestBody:
        content:
          '*/*':
            schema:
              type: string
```

1.1.26 generates
```
      summary: test Request
      operationId: testRequest
      requestBody:
        content: {}
```

For comparison 1.1.31 generates
```
      summary: test Request
      operationId: testRequest
      requestBody:
        content: {}
```

So content in the output is not being generated, but on looking at the issues fixed between the 2 versions, I'm not able to see an obvious cause.

**Comment by Stexxen** [comment]
I think it's related to this commit
https://github.com/springdoc/springdoc-openapi/commit/da51f64facd924dd1a71c6ee358af7c90fec175d#diff-516afd0f6db694c945f5c6214d2f66abL128-L133
The original code performed an addMediaType when the allConsumes array was empty, the new code does not.

However re-inserting that line for an empty array would not fix it as it is hardcoded to use MediaType. ALL_VALUE and doesn't take into account your fixes for #62

**Comment by Stexxen** [comment]
More info. 
I am able to get the correct functionality as long as I have a @RequestMapping at the class level that has a consumes attribute.

i.e.
```
@RestController("ProtoController")
@RequestMapping(path = "/api/v1/proto", consumes = {MediaType.APPLICATION_JSON_VALUE})
public class ProtoControllerImpl implements TestController {
```
Will generate correct values in the output.

Missing the consumes attribute causes the output to produce empty content.
It will also not work if it is set only at the method level, it has to be at the class level

**Comment by springdoc** [comment]
Hi, 

 Thank you for your feedback.
 A new enhancement will be available on v1.1.32 of springdoc-openapi.
